assets/newspack-ui/scss/elements/_index.scss

```diff
@@ -0,0 +1,25 @@
+@use 'boxes';
+@use 'forms';
+@use 'icons';
+@use 'tables';
+@use 'typography';
+
+// TODO: Find a better home for this:
```

Comment:
I don't mind having misc in the index file. Alternatively, we could go with a `misc.scss`. WDYT?

Reply:
Or even a `misc/` directory with each misc feature as its own `word-divider.scss` file. It helps organizing and forces us to think twice before adding more "misc" to the mix 😅

Reply:
`/misc` directory added in [d81a167](https://github.com/Automattic/newspack-plugin/pull/2810/commits/d81a16709a64af9cae5fb89564ebdbc5acfd8c8b) 🙂